examples/logging/README.md

```diff
@@ -0,0 +1,340 @@
+# DeepSparse Logging
+
+DeepSparse Logging provides operational teams with access to the telemetry needed to monitor a deployment. 
+
+For users seeking to put ML into production, these data are the raw materials that underpin the monitoring processes needed to create a system with consistently fresh and accurate predictions.
+
+<p align="center">
+     <img src="continual-learning.png"
+          alt="Continual Learning Diagram"
+          width="50%"
+     />
+</p>
+
+There are many types of downstream monitoring tasks that ML teams may want to perform. Some are easier to do (such as looking at system performance) and some are harder (looking at model accuracy requires manually labeling some data afterwards). Examples include:
+- **System performance:** what is the latency/throughput of a query?
+- **Data quality:** is there an error in the model pipeline?
```

Comment:
I don't think the error in the model pipeline is a data quality problem. What do we want to say here?

Reply:
Model pipeline shouldn't be the word. I was thinking something more like the camera is broken or something that causes all 0s to be sent to the model. I will update